back/app/models/concerns/participation_context.rb

```diff
@@ -52,10 +52,10 @@ module ParticipationContext
       end
       validates :upvoting_limited_max, presence: true,
         numericality: { only_integer: true, greater_than: 0 },
-        if: %i[ideation_or_budgeting? upvoting_limited?]
+        if: %i[can_contain_ideas? upvoting_limited?]
```

Comment:
Although with the current implementation of the method `can_contain_ideas?` this is correct, it does not express intent and it is not future proof. What's intended here is "is voting allowed?" In the future, when we will have more participation methods, e.g. Q&A, then voting will be supported for some of them. "can_contain_ideas?" will not express that. The code before your change did not express that either.
So for now this change is ok, but we will have to change this in the future.